src/lib/exposes.ts

```diff
@@ -1134,6 +1134,14 @@ export const presets = {
             )
             .withValueMin(0)
             .withValueMax(35),
+    max_temp: () =>
+        new Numeric('max_temp', access.STATE_SET)
```

Comment:
`max_temperature_limit` to be in sync with the `min_temperature_limit` below

Reply:
check lines 1127 and 1129

Reply:
Can you update `moes_thermostat_max_temperature_limit` instead? Use the correct datapoint

Reply:
it is 18 but needed 19

Reply:
Different points for most devices and this one

```
                device?.manufacturerName === '_TZE204_aoclfnxz'
                    ? e.max_temp().withValueMin(45).withValueMax(70)
                    : e.max_temperature_limit().withValueMax(45),
```

Reply:
But you can do an `if/else` in `moes_thermostat_max_temperature_limit`:

```
if (device?.manufacturerName === '_TZE204_aoclfnxz') {
  // send to 19
} else {
  // send to 18
}
```

Reply:
Two layers of checking manufacturer name looks like layered spaghetty ))

But I'll look to it, ok

Reply:
Ok but then reasonable to do the same with deadzone